Sources/AsyncHTTPClient/ConnectionPool/HTTP2/HTTP2Connection.swift

```diff
@@ -164,15 +164,23 @@ final class HTTP2Connection {
         return promise.futureResult
     }
 
-    private func start() -> EventLoopFuture<Int> {
+    func start0() -> EventLoopFuture<Int> {
```

Comment:
Nit: should we give this an underscore?

Reply:
Added in https://github.com/swift-server/async-http-client/pull/646/commits/510d6355ea9820d5509e93e428f42e3918479693